osquery/logger/plugins/tests/logger_plugin_util_tests.cpp

```diff
@@ -0,0 +1,209 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <chrono>
+#include <thread>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <osquery/dispatcher.h>
+#include <osquery/logger.h>
+
+#include "osquery/core/test_util.h"
+#include "osquery/logger/plugins/logger_plugin_util.h"
+
+namespace osquery {
+
+using namespace testing;
+
+class LoggerPluginUtilTests : public Test {
+ public:
+  void SetUp() {
```

Comment:
Ditch this override. `initTesting()` should be called once, in the tester's `main()`.